kolibri/plugins/learn/assets/src/vue/explore-page/index.vue

```diff
@@ -69,8 +91,133 @@
 <style lang="stylus" scoped>
 
   @require '~core-theme.styl'
-
-  .topic-page
-    padding-left: 10px
+  $thumbnail-width = 200
+  $margin-width = 24
+  card-list-container-width(n-cols)
+    width: ($thumbnail-width * n-cols) + ($margin-width * n-cols)
+  card-list-container-max-width(n-cols)
+    max-width: ($thumbnail-width * n-cols) + ($margin-width * n-cols)
+  input-width(n-cols)
+    width: $thumbnail-width * (n-cols - 1)
+  media-query(n-cols)
+    .page-container
+      max-width: ($thumbnail-width * n-cols) + $margin-width * (n-cols - 1)
+    .card-list-container
+      max-width: ($thumbnail-width * n-cols) + $margin-width * n-cols
+    .tool-bar-container
+      width: ($thumbnail-width * n-cols) + $margin-width * (n-cols - 1)
+    .tool-bar-container .search-input-active
+      width: $thumbnail-width * (n-cols) * 0.7
+    .tool-bar-container .tool-bar-center
+      -webkit-transform: translateX(-(((($thumbnail-width * n-cols) + $margin-width * (n-cols - 1)) - ($thumbnail-width * (n-cols) * 0.7))/2)px)
+  .section-title
+    margin-top: 5vh
+    font-size: 1.2em
+    font-weight: 700
+  .tool-bar-container
+    position: fixed
+    top: 0
+    card-list-container-width(1)
+    height: 30px
+    padding: 30 0
+    background-color: $core-bg-canvas
+    text-align: center
+    z-index: 1
+    .breadcrumbs-container
```

Comment:
Looks like some of the classes aren't used. Would you mind deleting all unused styles?